### ISSUE TITLE
Issue with /docs/getting-started-guides/centos/centos_manual_config/

### ISSUE THREAD

**Issue by dbafast** [kubernetes/kubernetes.github.io#3586]
**This is a...** 

- [ ] Feature Request
- [x] Bug Report

**Problem:**
Failed stock install
Installed Linode server with Centos 7 image,
Linux li773-70.members.linode.com 4.9.15-x86_64-linode81 #1 SMP Fri Mar 17 09:47:36 EDT 2017 x86_64 x86_64 x86_64 GNU/Linux
#############################
 pulled docker from here 
echo "[virt7-docker-common-release]
name=virt7-docker-common-release
baseurl=http://cbs.centos.org/repos/virt7-docker-common-release/x86_64/os/
gpgcheck=0" >> /etc/yum.repos.d/virt7-docker-common-release.repo
#############################
[root@li773-70 ~]# systemctl start docker
A dependency job for docker.service failed. See 'journalctl -xe' for details.
[root@li773-70 ~]# journalctl -xe
-- Defined-By: systemd
-- Support: http://lists.freedesktop.org/mailman/listinfo/systemd-devel
--
-- Unit docker-cleanup.service has finished starting up.
--
-- The start-up result is done.
Apr 29 16:48:40 li773-70.members.linode.com docker-storage-setup[16997]: INFO: Volume group backing root filesystem could not be determined
Apr 29 16:48:40 li773-70.members.linode.com flanneld-start[16996]: I0429 16:48:40.579718   16996 main.go:132] Installing signal handlers
Apr 29 16:48:40 li773-70.members.linode.com flanneld-start[16996]: I0429 16:48:40.580406   16996 manager.go:136] Determining IP address of default interface
Apr 29 16:48:40 li773-70.members.linode.com flanneld-start[16996]: I0429 16:48:40.580927   16996 manager.go:149] Using interface with name eth0 and address 104.200.24.70
Apr 29 16:48:40 li773-70.members.linode.com flanneld-start[16996]: I0429 16:48:40.582509   16996 manager.go:166] Defaulting external address to interface address (104.200.24.70)
Apr 29 16:48:40 li773-70.members.linode.com flanneld-start[16996]: E0429 16:48:40.583215   16996 network.go:102] failed to retrieve network config: client: etcd cluster is unavailable or mis
Apr 29 16:48:40 li773-70.members.linode.com docker-storage-setup[16997]: ERROR: No valid volume group found. Exiting.
Apr 29 16:48:40 li773-70.members.linode.com systemd[1]: docker-storage-setup.service: main process exited, code=exited, status=1/FAILURE
Apr 29 16:48:40 li773-70.members.linode.com systemd[1]: Failed to start Docker Storage Setup.

**Proposed Solution:**
This is a clean install of docker on a clean install of centos image. And it doesn't function. ??

**Page to Update:**
http://kubernetes.io/...

**Comment by dbafast** [comment]
And one more try USING CE instead of redhat distro 
INSTALL OF DOCKER CE DOES NOT WORK EITHER :
[root@li1433-168 ~]# yum list docker-ce.x86_64  --showduplicates |sort -r
 * updates: mirrors.linode.com
Loading mirror speeds from cached hostfile
Loaded plugins: fastestmirror
 * extras: mirrors.linode.com
docker-ce.x86_64            17.03.1.ce-1.el7.centos             docker-ce-stable
docker-ce.x86_64            17.03.0.ce-1.el7.centos             docker-ce-stable
 * base: mirrors.linode.com
Available Packages


Kubernetes installation conflicts with docker-ce-17.03.0            OPEN BUG


So basically Kubernetes on centos 7     does not function .  YIKES

**Comment by ahmetb** [comment]
@dbafast can you please open an issue on the Kubernetes main repository or ask on https://groups.google.com/forum/#!forum/kubernetes-users ? This issue tracker is particularly for documentation and I don’t think we can provide the required assistance.

**Comment by jaredbhatti** [comment]
Closing this issue since it's not a docs issue.  Please reopen if necessary.  Thanks!
